justfile

```diff
@@ -134,7 +134,17 @@ run-telemetry: devenv
 
 
 test-all *args: assets
-    COVERAGE_PROCESS_START="pyproject.toml" ./scripts/test-coverage.sh {{ args }}
+    #!/bin/bash
+    export COVERAGE_PROCESS_START="pyproject.toml"
+    export COVERAGE_REPORT_ARGS="--omit=jobserver/github.py,jobserver/opencodelists.py,tests/fakes.py,tests/verification/*"
+    ./scripts/test-coverage.sh -m "not verification" {{ args }}
+
+
+test-verification *args: assets
+    #!/bin/bash
+    export COVERAGE_PROCESS_START="pyproject.toml"
+    export COVERAGE_REPORT_ARGS="--include=jobserver/github.py,jobserver/opencodelists.py,tests/fakes.py,tests/verification/*"
+    ./scripts/test-coverage.sh -m "verification" {{ args }}
```

Comment:
Looking into a bit more detail about these tests, I think we have some integration tests as well as verification tests, so enabling coverage seems like it's the right thing to do. Although, as you say, we can always revisit that idea in future.